daffodil-core/src/main/scala/org/apache/daffodil/runtime1/SchemaSetRuntime1Mixin.scala

```diff
@@ -35,6 +35,17 @@ trait SchemaSetRuntime1Mixin { self : SchemaSet =>
 
   requiredEvaluationsAlways(parser)
   requiredEvaluationsAlways(unparser)
+  requiredEvaluationsAlways(root.elementRuntimeData.initialize)
+
+  /**
+   * This initialization is required for simpleTypeDefs used only
+   * for type calculations where those simpleTypeDefs do not have
+   * a corresponding element of that type.
+   */
+  requiredEvaluationsAlways(typeCalcMap.foreach{
+    case (_, typeCalculator) =>
+      typeCalculator.initialize()
+  })
 
```

Comment:
Arguably, the typeCalcMap ends up in a RuntimeData data structure such as the SchemaSetRuntimeData, so this initialization goes on that object's initialize method, and here we would just invoke that method.

Reply:
Not fixing for now.